arangod/GeneralServer/CommTask.h

```diff
@@ -87,10 +87,15 @@ class CommTask : public std::enable_shared_from_this<CommTask> {
 
   virtual void start() = 0;
   virtual void close() = 0;
-  
+
+    // returns the number of scheduled requests
+    std::size_t getRequestCount() const { return _requestCount; }
+
+    void setKeepAliveTimeoutReached() { _keepAliveTimeoutReached = true; }
```

Comment:
The indentation of the new code doesn't seem to match the indentation of the existing code around it.